busy_beaver/apps/upcoming_events/cards.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
+
+from busy_beaver.adapters.meetup import EventDetails
+from busy_beaver.toolbox.slack_block_kit import Context, Divider, Section
+
+
+class UpcomingEventList:
```

Comment:
Following the factory pattern to create composable objects we can render in Slack!